esx_service/utils/kvESX.py

```diff
@@ -35,9 +35,9 @@
 useSideCarCreate = False
 DVOL_KEY = "docker-volume-vsphere"
 
-# Results in a buffered, locked, filter-less open,
+# Results in a no-io, filter-less open,
 # all vmdks are opened with these flags
-VMDK_OPEN_FLAGS = 524312
```

Comment:
These are disklib flags - you can check opengrok.eng.vmware.com and search
for OPEN_LOCK (for example) in diskLibTypes.h. A few of those flags are put
together for the opens we do.

On Thu, Jun 16, 2016 at 10:47 PM, Ritesh H Shukla <notifications@github.com>
wrote:

> In esx_service/utils/kvESX.py
> <https://github.com/vmware/docker-volume-vsphere/pull/476#discussion_r67386408>
> :
>
> >  # all vmdks are opened with these flags
> > -VMDK_OPEN_FLAGS = 524312
>
> Is there any documentation that you can refer to for the value picked?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/vmware/docker-volume-vsphere/pull/476/files/49a5ca888603683a200a06d640fe5edf8c35d389#r67386408>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/APHseMJC6nHfw1659kiMgeImtDCUpwwxks5qMYUygaJpZM4I3SoA>
> .
>